x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/waterfall_chart_wrapper.test.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { fireEvent, waitFor } from '@testing-library/react';
+import { render } from '../../../../../lib/helper/rtl_helpers';
+import { WaterfallChartWrapper } from './waterfall_chart_wrapper';
+import { networkItems as mockNetworkItems } from './data_formatting.test';
+
+describe('WaterfallChartWrapper', () => {
```

Comment:
The waterfall chart wrapper is the best place to do unit tests that are as integrated from a front end perspective as a possible. From this parent component I can test the interaction of the click on the sidebar to opening the flyout. I have no idea how to test canvas clicks for actual clicks on the chart, and a bit of research didn't provide immediate results. Let me know if you have any ideas.